components/trading-simulation/simulation-performance.tsx

```
@@ -0,0 +1,219 @@
+          </Card>
+        </TabsContent>
+
+        <TabsContent value="trades">
+          <Card>
+            <CardHeader>
+              <CardTitle>Virtual Trade History</CardTitle>
+            </CardHeader>
+            <CardContent>
+              <div className="space-y-4">
+                {tradeHistory.map((trade, index) => (
+                  <div key={index} className="flex items-center justify-between p-4 rounded-lg border bg-card/50">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)